core/src/main/scala/org/bitcoins/core/wallet/utxo/UTXOSpendingInfo.scala

```diff
@@ -67,19 +67,22 @@ sealed abstract class UTXOSpendingInfo {
   * If you over-specify a path, such as giving a condition where none is needed,
   * then the remaining over-specified path will be ignored.
   *
-  * Note that we do not yet support nested conditionals.
-  *
   * For example, if you wanted to spend a ConditionalScriptPubKey(P2PK1, P2PK2)
   * (which looks like OP_IF <P2PK1> OP_ELSE <P2PK2> OP_ENDIF) with the P2PK1 case,
-  * then you would construct a ConditionalSpendingInfo using ConditionTrue as your
-  * ConditionalPath. Otherwise if you wanted to use P2PK2 you would use ConditionFalse.
+  * then you would construct a ConditionalSpendingInfo using conditionTrue as your
+  * ConditionalPath. Otherwise if you wanted to use P2PK2 you would use conditionFalse.
```

Comment:
nit: `P2PK2`

Reply:
not sure which part you are referring to?